microservice/package/src/main/java/com/cumulocity/agent/packaging/BaseMicroserviceMojo.java

```diff
@@ -255,4 +258,16 @@ private String getJavaVersion() {
         }
         return javaRuntime;
     }
+
+    protected String getTargetFilename(String targetArchitecture) {
+        if (targetArchitecture.equals(DockerBuildSpec.DEFAULT_TARGET_DOCKER_IMAGE_PLATFORM)) {
+            return getDefaultTargetFilename();
+        } else {
```

Comment:
There is no need for else statement as the block is always executed when there condition is not met